xgboost-predictor/src/main/java/biz/k11i/xgboost/learner/ObjFunction.java

```diff
@@ -86,6 +91,26 @@ public float predTransform(float pred) {
         return pred;
     }
 
+    /**
+     * Logistic regression.
+     */
+    static class RegObjFunction extends ObjFunction {
+        @Override
+        public float[] predTransform(float[] preds) {
+            if (preds.length != 1)
+                throw new IllegalStateException(
+                    "Regression problem is supposed to have just a single predicted value, got " + preds.length + " instead."
+                );
+            preds[0] = (float) Math.exp(preds[0]);
+            return preds;
+        }
+
+        @Override
+        public float predTransform(float pred) {
```

Comment:
```suggestion
        public float predTransform(final float pred) {
```

Nitpicking level 20000000000000000000000000000.